site/templates/items/meta_item.jet

```diff
@@ -31,7 +31,6 @@
   {{pillSupplement := ""}}
   {{slugRoute := routeToSlug(item.Slug)}}
   {{if isEpisode}}
-    {{slugRoute = routeToSlug(item.InnerItem.Season.Slug) + "#episode-" + item.InnerItem.EpisodeNumber}}
```

Comment:
We were making an episode-specific route here but has been delegated to kibble instead